### PR TITLE
setRemoteOperationListener is not documented

### PR DESCRIPTION
**Pull request by nikitag-ms**
* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [x] Did you check UI tests on the sample app? They are not executed on CI.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ ___
 ### App Center Data
 
 * **[Fix]** Fix an issue where invalid characters in the document ID are accepted at creation time but causing errors while trying to read or delete the document. The characters are `#`, `\`, `/`, `?`, and all whitespaces.
+* **[Feature]** `setRemoteOperationListener` method allows to be notified of a pending operation being executed when a client device goes from offline to online.
```
Comment (guperrot) [inline CHANGELOG.md]: Please update the corresponding release too.

Comment (nikitag-ms) [reply CHANGELOG.md]: Good catch. Updated https://github.com/microsoft/appcenter-sdk-android/releases/tag/2.1.0

Comment (guperrot) [reply CHANGELOG.md]: But wasn't it 2.0?

Comment (guperrot) [reply CHANGELOG.md]: yes it was already in 2.0, need to revert this

**Pull request merged**